assets/pages/contributions-landing/newContributionsLanding.scss

```diff
@@ -649,3 +650,82 @@ form {
   fill: inherit;
 }
 
+
+/*= THANK YOU PAGE */
+
+.hidden {
+  display: none;
+}
+
+.confirmation {
+  border-bottom: 1px solid gu-colour(garnett-neutral-4);
+  font-weight: 300;
+  margin: 0 -10px;
+  padding: 0 10px;
+}
+
+.confirmation__maintitle {
+  margin: 0;
+}
+
+.confirmation__amount {
+  align-items: center;
+  background: gu-colour(sport-garnett-media-main-1);
+  border-radius: 100%;
+  color: gu-colour(garnett-neutral-5);
+  display: flex;
+  float: right;
+  font: normal 16px/1 $gu-text-sans-web;
+  height: 50px;
+  justify-content: center;
+  margin: 0 0 0 ($gu-h-spacing / 2);
+  width: 50px;
+}
+
+.confirmation__title,
+.confirmation__message {
+  font: 300 22px/1.2 $gu-headline;
+  margin: ($gu-v-spacing / 2) 0 $gu-v-spacing;
+}
+
+.confirmation p {
+  margin: $gu-v-spacing 0 ($gu-v-spacing * 2);
+}
+
+.confirmation--newsletter {
+  background: url("/assets-payment/newsletters.png");
+  padding-bottom: 100px;
+}
+
+.confirmation--backtothegu {
+  margin-bottom: $gu-v-spacing + 20px;
+  padding-bottom: $gu-v-spacing;
+  padding-top: $gu-v-spacing;
+}
+
+.button {
+  align-items: center;
+  border: 1px solid currentColor;
+  border-radius: $gu-v-spacing * 2;
+  display: inline-flex;
+  font-family: $gu-text-sans-web;
+  height: $gu-v-spacing * 3.5;
+  line-height: 1;
+  padding: 0 $gu-h-spacing;
+  text-decoration: none;
+}
+
+.button--newsletter {
+  color: #29a05f;
```

Comment:
Should this be a `gu-colour`?

Reply:
There's none 😞  perhaps @Amohkhan can help?